IntegrationTests/Fixtures/XCBuild/TestProducts/Foo/Package.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
+    name: "Foo",
+    dependencies: [
+        .package(path: "../Bar")
+    ],
+    targets: [
+        .target(name: "FooLib", dependencies: [
+            .product(name: "BarLib", package: "Bar"),
+        ]),
+        .testTarget(name: "FooTests", dependencies: ["FooLib"]),
+        .testTarget(name: "CFooTests", dependencies: ["FooLib"]),
+    ],
+    swiftLanguageVersions: [.v4_2, .v5]
```

Comment:
Is it not possible to re-use existing fixtures?

Reply:
We could, but I purposefully configured those packages to test certain code paths of the PIF builder. What do you think?